Sources/NostrSDK/RelayResponse.swift

```diff
@@ -20,6 +20,7 @@ fileprivate struct EventKindMapper: Decodable {     // swiftlint:disable:this pr
         switch kind {
         case .setMetadata:      return SetMetadataEvent.self
         case .textNote:         return TextNoteEvent.self
+        case .directMessage:    return DirectMessageEvent.self
```

Comment:
Can we move this one down one position to keep them generally in kind number order?